internal/rbac/interactor.go

```diff
@@ -0,0 +1,35 @@
+package rbac
+
+import (
+	"context"
+
+	"github.com/SatorNetwork/sator-api/internal/jwt"
+)
+
+func CheckRole(userRole Role, requiredRoles ...Role) error {
```

Comment:
wanted to clarify some points:
- as I understand if `userRole` is equal to one of `requiredRoles` or `requiredRoles` contains `AvailableForAllRoles` we should be good?
- if above is correct, I'm not sure about `...` syntax, because this syntax implies that `requiredRoles` param is optional but actually it's required? Because `CheckRole(userRole)` always returns error?